plugins/chat/app/services/chat/publisher.rb

```diff
@@ -22,19 +41,38 @@ def self.publish_new!(chat_channel, chat_message, staged_id)
       content[:staged_id] = staged_id
       permissions = permissions(chat_channel)
 
-      MessageBus.publish(root_message_bus_channel(chat_channel.id), content.as_json, permissions)
+      message_bus_targets.each do |message_bus_channel|
+        MessageBus.publish(message_bus_channel, content.as_json, permissions)
+      end
 
-      MessageBus.publish(
-        self.new_messages_message_bus_channel(chat_channel.id),
-        {
-          channel_id: chat_channel.id,
-          message_id: chat_message.id,
-          user_id: chat_message.user.id,
-          username: chat_message.user.username,
-          thread_id: chat_message.thread_id,
-        },
-        permissions,
-      )
+      if chat_message.thread_reply?
+        MessageBus.publish(
+          root_message_bus_channel(chat_channel.id),
+          {
+            type: :update_thread_indicator,
```

Comment:
I would prefer a name which is not tied to the frontend implementation and more about what is happening here

Reply:
Yep will change based on my other comment too.